packages/eddsa-pcd/src/util/util.ts

```diff
@@ -1,5 +1,10 @@
 import { getRandomValues, toHexString } from "@pcd/util";
 
+/**
```

Comment:
Are we going to revert this comments too? @cedoor

Reply:
Oh right!